src/GitHub.App/Services/PullRequestService.cs

```diff
@@ -90,7 +90,7 @@ public class PullRequestService : IPullRequestService
                 if (readPullRequests == null)
                 {
                     readPullRequests = new Query()
-                          .Repository(Var(nameof(owner)), Var(nameof(name)))
+                          .Repository(Var(nameof(name)), Var(nameof(owner)))
```

Comment:
Rather than simply switching the order of these args, I'd prefer to use named arguments where possible.

Reply:
Fixed.